templates/components/agents/python/blog/app/agents/publisher.py

```
@@ -11,11 +11,11 @@ def get_publisher_tools() -> Tuple[List[FunctionTool], str, str]:
     tools = []
     # Get configured tools from the tools.yaml file
     configured_tools = ToolFactory.from_env(map_result=True)
-    if "document_generator" in configured_tools.keys():
-        tools.extend(configured_tools["document_generator"])
+    if "generate_document" in configured_tools.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)